testsuite/integration/smoke/pom.xml

```diff
@@ -36,13 +36,43 @@
         <jboss.dist>${basedir}/target/wildfly</jboss.dist>
 
     </properties>
-        <dependencies>
-            <dependency>
-                <groupId>${project.groupId}</groupId>
-                <artifactId>wildfly-galleon-pack</artifactId>
-                <type>pom</type>
-            </dependency>
-        </dependencies>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-core-client</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-jms-client</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-server</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.jms</groupId>
+            <artifactId>jboss-jms-api_2.0_spec</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.ws.rs</groupId>
+            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <!-- This is less than ideal, but required for some parsing tests.
+             See DataSourceOperationsUnitTestCase and AddMySqlDataSourceOperationsUnitTestCase
+        -->
```

Comment:
This should be fixed for sure. (I don't mean in this PR though.)

Tests of model parsing should be done in the subsystem's own module.

IMO DsMgmtTestBase.marshalAndReparseDsResources, ResourceAdapterOperationsUnitTestCase.marshalAndReparseRaResources and AbstractMgmtTestBase.modelToXml should all be removed.

Reply:
Makes sense to me. I'll file an issue for this to get fixed later. I might file an issue for the other places I think should be fixed and the component lead can decide how feasible that is.